tests/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generates a random lowercase ASCII string using the code points between 0 and
+ * 128.
```

Comment:
nit: It generates a random string of the lowercase latin letters, `[a-z]`. Code points 97-122.